tests/neg/i1641.scala

```diff
@@ -2,7 +2,7 @@ package bar { object bippy extends (Double => String) { def apply(x: Double): St
 package object println { def bippy(x: Int, y: Int, z: Int) = "(Int, Int, Int)" }
 object Test {
   def main(args: Array[String]): Unit = {
-    println(bar.bippy(5.5)) // error
+    println(bar.bippy(5.5))
```

Comment:
Isn't it concerning that this is now different from what scalac does? Is there some more subtlely in the way definitions are looked up that we're missing?